testProjectKotlinDslCopySpec/settings.gradle.kts

```diff
@@ -0,0 +1,8 @@
+pluginManagement {
+
+    repositories {
```

Comment:
This can be put directly into the `repositories{}` closure in the buildscript.

Reply:
ok will do